interface/themes/login_page.scss

```diff
@@ -28,12 +28,12 @@ body.login {
 }
 
 .login .container {
-  border-radius: 10px;
   background: $white;
+  border-radius: 10px;
+  box-shadow: 0 30px 60px 0 rgba($black, 0.3);
   margin-top: 75px;
```

Comment:
Something is wrong with Travis. Could you please explain how what Travis is reporting? 

![image](https://user-images.githubusercontent.com/834958/84769900-9c4cd600-afa4-11ea-8b18-234ea570dbc9.png)